src/blueprint/from_config.rs

```diff
@@ -278,7 +278,10 @@ fn update_http(field: &config::Field, b_field: FieldDefinition, config: &Config)
 
           Valid::Ok(b_field)
         }
-        None => Valid::fail("No base URL defined".to_string()),
+        None => {
+          std::process::exit(1);
```

Comment:
I think we need a more generic approach instead of exiting with status code 1 for this specific error.